sbncode/CAFMaker/FillReco.cxx

```diff
@@ -273,6 +283,38 @@ namespace caf
       else {
         srslice.nu_score = -1;
       }
+      // NeutrinoID (SliceID) features
+      if (properties.count("NuNFinalStatePfos")) {
+        srslice.nuid.NuNFinalStatePfos = properties.at("NuNFinalStatePfos");
+      }
+      if (properties.count("NuNHitsTotal")) {
+        srslice.nuid.NuNHitsTotal = properties.at("NuNHitsTotal");
+      }
+      if (properties.count("NuVertexY")) {
+	srslice.nuid.NuVertexY = properties.at("NuVertexY");
+      }
+      if (properties.count("NuWeightedDirZ")) {
+	srslice.nuid.NuWeightedDirZ = properties.at("NuWeightedDirZ");
+      }
+      if (properties.count("NuNSpacePointsInSphere")) {
+	srslice.nuid.NuNSpacePointsInSphere = properties.at("NuNSpacePointsInSphere");
```

Comment:
Something up with the indentation (literal tabs?) here

Reply:
Interesting. Pretty sure it looked okay in my editor but wouldn't be the first time indentations were treated weirdly in an editor 🤦‍♂️. I'll touch that up.